### PR TITLE
Bump node from 18.17.1 to 20.11.0

### PR DESCRIPTION
**Pull request by kyntk**
## What

- Bump node from 18.17.1 to 20.11.0.
- Fix rack response

### CODE REVIEW

File: server.coffee [hunk 1]
```diff
@@ -194,7 +194,7 @@ process_url = (url, transferredHeaders, resp, remaining_redirects) ->
 # decode a string of two char hex digits
 hexdec = (str) ->
   if str and str.length > 0 and str.length % 2 == 0 and not str.match(/[^0-9a-f]/)
-    buf = new Buffer(str.length / 2)
+    buf = new Buffer.alloc(str.length / 2)
```
Comment (kyntk) [inline server.coffee:197]: Fixed this error
```
DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ Unreleased
 
 <!-- Add descriptions of changes here -->
 
+* Add support for Node v20 and drop support for Node v17 [22](https://github.com/increments/camo/pull/22)
```
Comment (atm-snag2) [inline CHANGELOG.md]: maybe

```suggestion
* Add support for Node v20 and drop support for Node v17 or before [22](https://github.com/increments/camo/pull/22)
```

Comment (kyntk) [reply CHANGELOG.md]: @atm-snag2
Thanks! I fixed it 👍

**Review by atm-snag2 — APPROVED** [review]
LGTM

**Pull request merged**